sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -539,11 +539,10 @@ object ParquetFileFormat extends Logging {
       parFiles.flatMap { currentFile =>
         try {
           // Skips row group information since we only need the schema.
-          // ParquetFileReader.readFooter throws RuntimeException, instead of IOException,
+          // ParquetUtils.readFooter throws RuntimeException, instead of IOException,
           // when it can't read the footer.
           Some(new Footer(currentFile.getPath(),
```

Comment:
We can replace this here since it looks causing resource leak. I filed a JIRA https://issues.apache.org/jira/browse/PARQUET-1368